test/helpers.go

```diff
@@ -68,6 +68,7 @@ func createStubbedPrepareCmd(cs *CmdStubber) func(*exec.Cmd) run.Runnable {
 		if call >= len(cs.Stubs) {
 			panic(fmt.Sprintf("more execs than stubs. most recent call: %v", cmd))
 		}
+		// fmt.Printf("Called stub for `%v`\n", cmd) // Helpful for debugging
```

Comment:
This makes me thing that should expand our cmd stubber to be able to assert that a specific command was invoked

Reply:
I think that would still run into similar problems. When things don't match up correctly you still want to print out the commands that are called. I think there is a better solution to what we're doing with command stubbing, but I've not been able to think of it yet!